tests/utils.go

```diff
@@ -1809,6 +1813,58 @@ func createNamespaces() {
 	}
 }
 
+func AddLabelToNamespace(namespace, key, value string) {
+	virtCli, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	ns, err := virtCli.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	old, err := json.Marshal(ns)
+	Expect(err).ToNot(HaveOccurred())
+	new := ns.DeepCopy()
+	if new.Labels == nil {
+		new.Labels = map[string]string{}
+	}
+	new.Labels[key] = value
+
+	newJson, err := json.Marshal(new)
+	Expect(err).ToNot(HaveOccurred())
+
+	patch, err := strategicpatch.CreateTwoWayMergePatch(old, newJson, ns)
+	Expect(err).ToNot(HaveOccurred())
+
+	_, err = virtCli.CoreV1().Namespaces().Patch(context.Background(), ns.Name, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func RemoveLabelFromNamespace(namespace, key string) {
+	virtCli, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	ns, err := virtCli.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	old, err := json.Marshal(ns)
+	Expect(err).ToNot(HaveOccurred())
+
+	new := ns.DeepCopy()
+	if new.Labels == nil {
+		// There are no labels, nothing to do
+		return
+	}
+	delete(new.Labels, key)
+
+	newJson, err := json.Marshal(new)
+	Expect(err).ToNot(HaveOccurred())
+
+	patch, err := strategicpatch.CreateTwoWayMergePatch(old, newJson, ns)
+	Expect(err).ToNot(HaveOccurred())
+
+	_, err = virtCli.CoreV1().Namespaces().Patch(context.Background(), ns.Name, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
+	Expect(err).ToNot(HaveOccurred())
+}
+
```

Comment:
Is there an alternative where we can put this in the `test/network` package ? 

I'd rather not bloat the utils file with more stuff, especially if it can be avoided.

Reply:
Before I go ahead and start being ceative, what do you think about `tests/network/utils/namespace.go`?

Reply:
Good enough for me. Let someone pitch in w/ the naming before moving it if you prefer.

Reply:
I want to correct myself; I actually meant something on [libnet](https://github.com/kubevirt/kubevirt/tree/1d7798a572eedcdb35f3292104006dd29db8dfe3/tests/libnet)

Reply:
Moved to `tests/libnet/namespace.go`